### PR TITLE
IAM&VM: IAM user feature & free_VM

### PR DESCRIPTION
**Pull request by rosasu**
R Title Format
IAM, user : add a new function, user
VM, free：change the content

What type of PR is this?
/Add Features
/change content

Which issue(s) this PR fixes?
na

What this PR does / why we need it ?
na

Special notes for your reviewer?
/assign @pethu1110

### CODE REVIEW

File: content/authorization/iam/manual/user.md [hunk 1]
```diff
@@ -19,10 +19,26 @@ weight: 40
...
+    >
+    >支持批量创建用户。
+
 4. 创建完成。
 
+    <img src="../../_images/user3_1.png" alt="图片" style="zoom:25%;" />
+
     <img src="../../_images/user3.png" alt="图片" style="zoom:25%;" />
 
+    >说明
```
Comment (pethu1110) [inline content/authorization/iam/manual/user.md:32]: “说明”加粗

File: content/compute/vm/faq/trial/free_instance.md [hunk 2]
```diff
@@ -26,6 +26,62 @@ https://www.qingcloud.com/consultation
 
 3. 优惠券使用
 
+<<<<<<< HEAD
```
Comment (pethu1110) [inline content/compute/vm/faq/trial/free_instance.md:29]: 这里？

File: content/authorization/iam/manual/user.md [hunk 3]
```diff
@@ -149,7 +183,17 @@ weight: 40
 
    <img src="../../_images/user19.png" style="zoom:40%;" />
 > 说明：
```
[the remark this answers is not in the document]
Comment (rosasu) [reply content/authorization/iam/manual/user.md:185]: 已修改

File: content/authorization/iam/manual/user.md (hunk 1 above)
Comment (rosasu) [reply content/authorization/iam/manual/user.md:32]: 已修改

File: content/compute/vm/faq/trial/free_instance.md (hunk 2 above)
Comment (rosasu) [reply content/compute/vm/faq/trial/free_instance.md:29]: 已修改

**Review by pethu1110 — APPROVED** [review]

**Pull request merged**